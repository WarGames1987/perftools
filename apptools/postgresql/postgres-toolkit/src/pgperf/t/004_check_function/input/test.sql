SELECT pgperf._check_function('_check_function');

SELECT pgperf._check_function('nosuchfunction');
