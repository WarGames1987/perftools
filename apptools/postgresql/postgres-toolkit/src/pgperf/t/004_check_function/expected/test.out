SELECT pgperf._check_function('_check_function');
 _check_function 
-----------------
 t
(1 row)

SELECT pgperf._check_function('nosuchfunction');
 _check_function 
-----------------
 f
(1 row)

